modules/ROOT/pages/dataweave-formats-avro.adoc

```diff
@@ -118,7 +118,7 @@ This format accepts properties that provide instructions for reading input data.
 [cols="1,1,1,3a", options="header"]
 |===
 |Parameter |Type |Default|Description
-|`schemaUrl` (Required) |`String`|`null`|The URL for the Avro schema. Valid URI schemes are `classpath://`, `file://`, or `http://`.
+|`schemaUrl` (Required) |`String`|`''`|The URL for the Avro schema. Valid values are `classpath://`, `file://`, or `http://`.
```

Comment:
Check if default remains `null` or is `''`.

Ok to say "Valid values are" now instead of "Valid URI schemes are"?